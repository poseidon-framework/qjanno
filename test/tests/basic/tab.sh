qjanno -t --raw --noOutHeader "SELECT * FROM data/tab.csv"
