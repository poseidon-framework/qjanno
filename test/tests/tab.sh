qhs -t -T -H "SELECT * FROM tab.csv"
