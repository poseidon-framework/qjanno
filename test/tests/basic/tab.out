data/tab.csv	a0	1	a2
data/tab.csv	b0	3	b2
data/tab.csv	c0		c2
